### PR TITLE
Blog: Hacktoberfest 2024

### PR DESCRIPTION
**Pull request by alex-fusionauth**

### CODE REVIEW

File: astro/src/content/blog/2024-hacktoberfest.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+description: FusionAuth joins in the Hacktoberfest event in 2024
+authors: Alex Patterson
+image: /img/blogs/2024-hacktoberfest/hacktoberfest-2024.png
+categories: News
+tags: github, hackathon, open source, hacktoberfest, javascript
+excerpt_separator: "{/* more */}"
+---
+import {RemoteCode} from '@fusionauth/astro-components';
+
+We're excited to announce FusionAuth's participation in this year's Hacktoberfest! Hacktoberfest is a global event that takes place every Autumn, during which developers are encouraged to contribute to open-source projects like FusionAuth. The goal is to foster a vibrant open-source community, celebrate shared knowledge, and make the world of coding more accessible to all. You can find other participating repositories by searching <a href="https://github.com/topics/hacktoberfest" target="_blank" rel="noopener noreferrer">GitHub</a> or <a href="https://gitlab.com/explore/projects/topics/hacktoberfest" target="_blank" rel="noopener noreferrer">GitLab</a>.
```
Comment (mark-robustelli) [inline astro/src/content/blog/2024-hacktoberfest.mdx:13]: Need to be careful here as FusionAuth is not open-source.  FusionAuth the company does have open-source projects. It is important to make that distinction.

File: astro/src/content/blog/2024-hacktoberfest.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+{/* more */}
+
+## Getting Started
+
+The first step is to <a href="https://hacktoberfest.com/register" target="_blank" rel="noopener noreferrer"> register for Hacktoberfest</a>. Once you've registered, any PRs submitted to FusionAuth (or any other repo with the label #Hacktoberfest) will count towards your participation in the event!
```
Comment (mark-robustelli) [inline astro/src/content/blog/2024-hacktoberfest.mdx:19]: Anyone that does not know what a PR is will probably not participate, but may want to spell out pull-request here for those reading.

**Review by mark-robustelli — APPROVED** [review]
A few things to think about, if you do make changes, let me know and I will approve again right away.

File: astro/src/content/blog/2024-hacktoberfest.mdx (hunk 1 above)
Comment (alex-fusionauth) [reply astro/src/content/blog/2024-hacktoberfest.mdx:13]: Think we are good here, same as last year. We always talk about how everything but the core is open-source 😄

**Pull request merged**